packages/main/src/Select.js

```diff
@@ -545,7 +562,13 @@ class Select extends UI5Element {
 		this._lastSelectedOption = this.options[this._selectedIndex];
 	}
 
+	_afterOpen() {
+		this.opened = true;
+		this._applyFocusAfterOpen()
```

Comment:
is this needed since we have fake focus?

Reply:
Correct, this is no longer needed as fake focus is completely handled by _syncSelection function.